src/App.js

```diff
@@ -1,23 +1,31 @@
 import React from "react";
 import "./App.css";
 import IconContainer from "./components/IconContainer";
-import { Switch, Route } from "react-router-dom";
+import { BrowserRouter, Switch, Route } from "react-router-dom";
```

Comment:
There is **BrowserRouter** inside the _index.js_. Did you see that? If these are the same you can delete this one.

Reply:
Yes they are the same, I will delete this one in the App.js

Reply:
We prefer to keep `BrowserRouter` in `App.js` for two reasons:

- it makes sense, as the `Switch` is in `App.js`
- tests complaint, as `BrowserRouter` and `Switch` are meant to be contiguous.

That is: it works both ways, but this one is preferrable